apps/storefront/vite.config.ts

```diff
@@ -103,31 +103,34 @@ export default defineConfig(({ mode }) => {
             const { name } = info;
             return name.includes('headless') ? '[name].js' : '[name].[hash].js';
           },
+          manualChunks: {
+            reactVendor: ['react', 'react-dom'],
+            intl: ['react-intl'],
+            mui: ['@mui/material'],
+            muiIcon: ['@mui/icons-material'],
+            redux: ['react-redux'],
+            dateFns: ['date-fns'],
+            lang: ['@b3/lang'],
+            pdfobject: ['pdfobject'],
+            resizable: ['react-resizable'],
+            pdf: ['react-pdf'],
+            toolkit: ['@reduxjs/toolkit'],
+            form: ['react-hook-form'],
+            router: ['react-router-dom'],
+            lodashEs: ['lodash-es'],
+            dropzone: ['react-dropzone'],
+            draggable: ['react-draggable'],
+            eCache: ['@emotion/cache'],
+            eReact: ['@emotion/react'],
+            eStyled: ['@emotion/styled'],
```

Comment:
do we need to put `redux` and `toolkit` into one trunk?  So `dropzone` and `draggable` do.

Reply:
I think it is necessary because the main entry file uses redux and toolkit. After packaging, the main entry file is too large and needs to be split